python/servo/package_commands.py

```diff
@@ -485,7 +501,17 @@ def upload_to_github_release(platform: str, package: str, package_hash: str) ->
             nightly_repo = g.get_repo(os.environ["NIGHTLY_REPO"])
             release = nightly_repo.get_release(github_release_id)
 
-            asset_name = f"servo-latest.{extension}"
+            if platform != "mac-arm64":
+                # Legacy assetname. Will be removed after a period with duplicate assets.
+                asset_name = f"servo-latest.{extension}"
+                package_hash_fileobj = io.BytesIO(f"{package_hash}  {asset_name}".encode("utf-8"))
+                release.upload_asset(package, name=asset_name)
+                # pyrefly: ignore[missing-attribute]
+                release.upload_asset_from_memory(
+                    package_hash_fileobj, package_hash_fileobj.getbuffer().nbytes, name=f"{asset_name}.sha256"
+                )
+            asset_platform = map_platform(platform)
+            asset_name = f"servo-{asset_platform}.{extension}"
```

Comment:
As long as we are creating a new schema, maybe we want to include the date in the name? I'm fine with this schema as well.